charts/v3.0.0/csi-driver-nfs/templates/csi-nfs-driverinfo.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1beta1
+kind: CSIDriver
+metadata:
+  name: nfs.csi.k8s.io
+spec:
+  attachRequired: false
+  volumeLifecycleModes:
+    - Persistent
+  podInfoOnMount: true
```

Comment:
What does the driver do with pod information?

Reply:
would be fixed by https://github.com/kubernetes-csi/csi-driver-nfs/pull/193